### ISSUE TITLE
Feature Request: Favorite music from control bar.

### ISSUE THREAD

**Issue by LI3GE** [eltoncezar/Spofy#7]
Hey,  I use the radio and playlist feature a lot in spotify to find new music and I'd love to to be able to favorite a song that I like without needing to go into spotify.  It'd mimic the cross in the regular application and send music to your songs list.

I know this is a QOL issue but I thought I'd put in a request anyway. I love the app and thanks for the hard work!

**Comment by eltoncezar** [comment]
Hi, thanks for your feedback. I thought no one was using it besides me :smiley: 
It's always nice to hear new ideas, since I created this app for my personal use, and there's a lot of room for improvement.
This is a feature I'm missing too, so I'm researching the best way to implement it. Let's see if I can make it work in a future version.
Again, thanks for your support, and I looking forward to hear your ideas!

**Comment by beppe9000** [comment]
Is the normal or local API capable of doing that?

**Comment by eltoncezar** [comment]
@beppe9000 yes, it is possible through the SpotifyWebAPI. I've already started this in https://github.com/eltoncezar/Spofy/commit/e91b739ce5605e64888a356ffa6a1de56e6cd449, if you want to take a look :)
